src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```diff
@@ -209,8 +374,38 @@ private static ProjectItemGroupElement CreateItemGroup(Project project, string f
             return itemGroup;
         }
 
+        /// <summary>
+        /// Adding version metadata to a given project item element.
+        /// </summary>
+        /// <param name="libraryDependency">Package Dependency of the package to be added.</param>
+        /// <param name="item">The item that the version metadata should be added to.</param>
+        /// <param name="versionAttribute">Boolean that expresses if the version attribute already exists or not.</param>
+        /// <returns>The package version that is added in the metadata.</returns>
+        private string AddVersionMetadata(LibraryDependency libraryDependency, ProjectItemElement item, ProjectMetadataElement versionAttribute)
```

Comment:
IMHO, this method should not accept `versionAttribute` parameter. The reason is we can identify the presence of `Version` attribute from the `item` parameter value. Please correct me if I am wrong.

Reply:
You are correct and I agree, it has been removed.